source/subtract.d.ts

```diff
@@ -1,4 +1,6 @@
-import type {NumberAbsolute, BuildTuple, And, Or} from './internal';
+import type {
+	NumberAbsolute, BuildTuple, And, Or,
```

Comment:
These should be on separate lines.

Reply:
Right, any idea why `xo` isn't picking up on that?

Reply:
No idea. First time I noticed it.

Reply:
Caused by https://github.com/xojs/eslint-config-xo/pull/83, which requires a newline in beginning and end when there are so many parameters, but doesn't seems to enforce it in-between them 🤔

Reply:
Fix is to modify it and remove the `minProperties`, one can then add `object-property-newline` to force newlines, but that will force it inbetween all properties and as such be equal to `minProperties: 1`, and its `allowAllPropertiesOnSameLine` allows all properties on a single line even when there's a curly newline, so will add this here as a workaround for now and then we can look at upstreaming to `xo`:

```json
	"object-curly-newline": ["error", {
		"multiline": true,
		"consistent": true
	}]
```

Reply:
We still want to only enforce the object to be on newlines when there are minimum 4 properties though. Not sure what the correct config would be for that.

Reply:
Now it only enforces it if there's a newline somewhere within the object